FirmataMarshaller.cpp

```diff
@@ -168,6 +168,8 @@ const
   if ( (Stream *)NULL == FirmataStream ) { return; }
   FirmataStream->write(START_SYSEX);
   FirmataStream->write(REPORT_FIRMWARE);
+  FirmataStream->write(firmata::FIRMWARE_MAJOR_VERSION);
```

Comment:
I still don't understand. These bytes aren't specified in the protocol for the query.